arch/risc-v/src/mpfs/mpfs_emmcsd.c

```diff
@@ -782,12 +782,22 @@ static void mpfs_sendfifo(struct mpfs_dev_s *priv)
        * come back when we're good to write again.
        */
```

Comment:
This is used with poll-mode, but by default, DMA is enabled (on bootloader and hart2 NuttX as well)?

Reply:
Good point. Looks like at least bootloader case there are send operations with interrupt (poll) mode, but also dmasendsetup is called. I'm wondering if there are overlapping DMA and interrupt based transfers.... need to check this!

Reply:
So, interrupt mode writes are normal in USB mass storage write case.